test/inductor/test_aot_inductor.py

```diff
@@ -394,6 +394,50 @@ def forward(self, x):
         example_inputs = (torch.rand(6, device=self.device),)
         self.check_model(Model(), example_inputs)
 
+    @unittest.skip("Skip this test, only for local test. SIGABRT is produced.")
```

Comment:
Can we use `signal.signal()` to catch `SIGABRT`? https://docs.python.org/3/library/signal.html#signal.signal

Reply:
Unfortunately no :( 
It's still a core-dump, I tried to rehook `SIGABRT` but still doesn't work. I don't know the reason, but from some stackoverflow that I read, it seems like I need to hook it with ctypes at the very moment of exec (which, in our case seems to be `AOTModelRunner`) and that would be a huge change and non-trivial with our current testing setup. This isn't a test that must-not break, since it's mostly a debugging feature, so it should be fine.